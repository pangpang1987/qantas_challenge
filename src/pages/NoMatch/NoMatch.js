import React from "react";

const NoMatch = () => (
  <div style={{textAlign: 'center'}}>
    <h2>404</h2>
    <h3>Sorry, but the page you are looking for is not exsited.</h3>
  </div>
);

export default NoMatch;
