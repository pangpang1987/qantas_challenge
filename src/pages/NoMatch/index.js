import React from 'react';

const NoMatch = () => {
  return (<div>NoMatch</div>);
}

NoMatch.propTypes = {

};

export default NoMatch;
