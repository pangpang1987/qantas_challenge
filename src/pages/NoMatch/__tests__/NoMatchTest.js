import React from "react";
import NoMatch from "../NoMatch";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<NoMatch page="http://www.facebook.com">Facebook</NoMatch>).toJSON();
  expect(tree).toMatchSnapshot();
});
