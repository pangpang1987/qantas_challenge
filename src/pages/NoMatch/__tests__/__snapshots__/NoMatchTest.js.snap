// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h2>
    404
  </h2>
  <h3>
    Sorry, but the page you are looking for is not exsited.
  </h3>
</div>
`;
